examples/validating-public-input/contracts/.env.example

```diff
@@ -0,0 +1,3 @@
+RPC_URL= #You can use publicnode RPC: https://ethereum-holesky-rpc.publicnode.com
+PRIVATE_KEY=
+ALIGNED_SERVICE_MANAGER_ADDRESS= #0x58F280BeBE9B34c9939C3C39e0890C81f163B623 for Holesky
```

Comment:
```suggestion
RPC_URL=<rpc_url> #You can use publicnode RPC: https://ethereum-holesky-rpc.publicnode.com
PRIVATE_KEY=<private_key>
ALIGNED_SERVICE_MANAGER_ADDRESS=<service_manager_address> #0x58F280BeBE9B34c9939C3C39e0890C81f163B623 for Holesky
```

Reply:
Solved in [c822542](https://github.com/yetanotherco/aligned_layer/pull/648/commits/c82254288832f0258e72d0fcaa71b617a9a73b16)